packages/server/src/fhir/search.ts

```diff
@@ -280,10 +270,10 @@ async function getSearchEntries<T extends Resource>(
   repo: Repository,
   searchRequest: SearchRequestWithCountAndOffset<T>,
   builder: SelectQuery
-): Promise<{ entry: BundleEntry<T>[]; rowCount: number; hasMore: boolean }> {
+): Promise<{ entry: BundleEntry<T>[]; rowCount: number; nextResource?: T }> {
   const rows = await builder.execute(repo.getDatabaseClient(DatabaseMode.READER));
   const rowCount = rows.length;
-  const resources = rows.slice(0, searchRequest.count).map((row) => JSON.parse(row.content as string)) as T[];
+  const resources = rows.slice(0, searchRequest.count + 1).map((row) => JSON.parse(row.content as string)) as T[];
```

Comment:
Why `slice()` here?  We request `+ 1` to know if there are more.  I think the `slice` is a no-op?